### DISCUSSION TITLE
3.0 timeline

### DISCUSSION THREAD

**Discussion by VladislavAntonyuk** [Blazor-Diagrams/Blazor.Diagrams#223] [Q&A] [answered]
Are there any estimates when version 3.0 will be ready for release?

are there any plans to publish preview versions?

**Comment by zHaytam** [comment, marked as answer]
Hello,

I'm aiming for previews to start once all the IMPORTANT features are done, probably the second week of September (8 to 14).
For the official release, most likely the last week of September or the start of October (depending on how the previews prove).

I'll release a preview and start building a personal app, that will help me validate all the features and everything.
Hope this answers you!

**Reply by VladislavAntonyuk** [reply]
thank you, you did great work! Looking forward to your previews.

**Comment by zHaytam** [comment]
https://github.com/Blazor-Diagrams/Blazor.Diagrams/releases/tag/3.0.0
